Gnuplot commit 4bcc37 Support for bulleted lists.
This fix comes from upstream in the development version of 
gnuplot 5.3.

diff -u docs/doc2texi.el.ORG docs/doc2texi.el
--- gnuplot-5.2.2/docs/doc2texi.el      2017-06-30 15:47:02.412241000 +0900
+++ gnuplot-5.2.2/docs/doc2texi.el      2017-06-30 15:47:27.938264000 +0900
@@ -760,8 +760,8 @@
     (while (re-search-forward "^#end" (point-max) "to_end")
       (replace-match "@end itemize" t))
     (goto-char (point-min))
-    (while (re-search-forward "^#b" (point-max) "to_end")
-      (replace-match "@item\n" t))
+    (while (re-search-forward "^#b " (point-max) "to_end")
+      (replace-match "@item\n " t))
     (goto-char (point-min))
     (while (re-search-forward "^##" (point-max) "to_end")
       (replace-match "" t))))
